### PR TITLE
Transition to Cartesi application templates

### PR DESCRIPTION
**Pull request by tuler**
This branch will be the `main` branch of the `cartesi/application-templates` repository.
Not to be merged here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Sunodo Templates
+# Cartesi Application Templates
 
-Template projects for Sunodo DApps. This repository is used by the Sunodo CLI to create new applications from a specified template project.
+Template projects for Cartesi DApps. This repository is used by the Cartesi CLI to create new applications from a specified template project.
```
Comment (endersonmaia) [inline README.md]:
```suggestion
Template projects for Cartesi Applications. This repository is used by the Cartesi CLI to create new applications from a specified template project.
```

**Review by endersonmaia — CHANGES_REQUESTED** [review]

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Sunodo
+Copyright (c) 2023 Cartesi
```
Comment (endersonmaia) [inline LICENSE]:
```suggestion
Copyright (c) 2024 Cartesi
```

**Review by endersonmaia — CHANGES_REQUESTED** [review]

**Review by endersonmaia — APPROVED** [review]
Remember to create a branch `0.4-sdk` pointing to main.

Shall we need to create pre-0.4 branches?

**Comment by tuler** [comment]
> Remember to create a branch `0.4-sdk` pointing to main.

`0.6` right?

> Shall we need to create pre-0.4 branches?

I don't think so.

**Comment by endersonmaia** [comment]
> > Remember to create a branch `0.4-sdk` pointing to main.
> 
> `0.6` right?
> 
> > Shall we need to create pre-0.4 branches?
> 
> I don't think so.

yes, `0.6-sdk` :man_facepalming:

**Comment by tuler** [comment]
Pushed to https://github.com/cartesi/application-templates

**Pull request closed without merging**